Britannia/events/BTWK_frontier_events.txt

```diff
@@ -531,7 +550,69 @@ BTWK_frontier_event.0204 = {
 	type = character_event
 	title = BTWK_frontier_event.0204
 	desc = {
-		desc = BTWK_frontier_event.0204_desc
+		desc = BTWK_frontier_event.0204_desc_header
+		first_valid = {
+			triggered_desc = { 
+				trigger = {
+					has_global_variable = fall_of_britannia
+				}
+				desc = BTWK_frontier_event.0204_desc_britannia_collapsed
+			}
+			triggered_desc = {
+				trigger = {
+					NOT = { has_global_variable = fall_of_britannia }
+				}
+				desc = {
```

Comment:
Might need to adjust the triggers a bit, I think you don't get anything if you're an Imperial-like culture in Caledonia

Reply:
Changed it to :
`
OR = {  # If in Caledonia and culture is NOT imperial-like, cannot receive this trigger
	NOT = {
		capital_province = {
			geographical_region = world_caledonia
		}
		is_imperial_like_culture_trigger = no
	}
	is_imperial_like_culture_trigger = yes
}
`
I think this should fix it? Hard to tell until Caledonia is added so I'll leave a note

Reply:
I thought that would've indented nicer. Let's try again:

OR = {  # Can't receive this desc if not imperial-like culture in Caledonia
	NOT = {
		capital_province = {
			geographical_region = world_caledonia
		}
		is_imperial_like_culture_trigger = no
	}
	is_imperial_like_culture_trigger = yes
}

Reply:
Sorry, which block would this be for? Or would it be a new one?

Reply:
Original block was:

NOT = {
	capital_province = {
		geographical_region = world_caledonia
	}
}
is_imperial_like_culture_trigger = yes

I'm replacing it with something that should fix the issue stated in your original comment.